### PR TITLE
Katie 404 page

### PR DESCRIPTION
**Pull request by kcbreuil**

### CODE REVIEW

File: web/src/components/index/indexFeatures.js [hunk 1]
```diff
@@ -95,9 +95,9 @@ export default function IndexFeatures({ content }) {
                     {features.map((feature) => {
                         return (
                             <GridItem
-                                key={feature.id}
-                                onClick={() => setActiveTab(feature.id)}
-                                isActiveTab={activeTab === feature.id}
+                                key={features.id}
+                                onClick={() => setActiveTab(features.id)}
+                                isActiveTab={activeTab === features.id}
```
Comment (asasmith) [inline web/src/components/index/indexFeatures.js]: I there a reason this is changed? Doesn't seem like it should be part of the work on the 404 page but maybe I'm missing something.

Also, I think it breaks the functionality. We're mapping over the `features` array using each `feature` item to return some jsx. `features` doesn't really exist inside the `map()` method.

File: web/src/pages/404.js [hunk 2]
```diff
@@ -1,63 +1,89 @@
...
-//     color: '#232129',
-//     fontFamily: '-apple-system, Roboto, sans-serif, serif',
-// };
-const headingStyles = {
-    marginTop: 0,
-    marginBottom: 64,
-    maxWidth: 320,
-};
+const StyledHeading = styled.h2`
+    ${tw`font-bold text-center text-5xl w-2/4 mb-8`}
```
Comment (asasmith) [inline web/src/pages/404.js]: Can you change `w-2/4` to `w-1/2`? It is minor but I think it reads cleaner.

File: web/src/pages/404.js [hunk 3]
```diff
@@ -1,63 +1,89 @@
...
-                <Link to="/">Go home</Link>.
-            </p>
-        </Main>
+        <Layout>
+            <Hero content={hero} />
+            <Wrapper>
+                <Img
+                    fixed={fourohfour.childImageSharp.fixed}
+                    style={{ marginTop: '-75px' }}
+                />
```
Comment (asasmith) [inline web/src/pages/404.js]: Convert this to a styled component and move a tailwind class for the negative margin.

File: web/src/pages/404.js [hunk 4]
```diff
@@ -1,63 +1,89 @@
...
+                <StyledHeading>
+                    Didn’t find what you’re looking for?
+                </StyledHeading>
+                <StyledP>
+                    This page doesn’t exist, click <Link to="/">here</Link> to
```
Comment (asasmith) [inline web/src/pages/404.js]: The designs show `click here` being wrapped in the `<Link>` component. They're also green and underlined. Convert the `<Link>` to a styled component and use tailwind classes to style.

File: web/src/pages/404.js [hunk 5]
```diff
@@ -1,63 +1,89 @@
...
+                    head back to the homepage{' '}
```
Comment (asasmith) [inline web/src/pages/404.js]: I don't think that `{' '}` is doing anything here. Please remove.

File: web/src/pages/404.js [hunk 6]
```diff
@@ -1,63 +1,89 @@
...
+            heading: heroHeading
+        }
+        cta: sanityAboutPage {
+            ctaHeading
+            link {
+                text
+                url
+            }
+        }
+        fourohfour: file(relativePath: { regex: "/404/" }) {
```
Comment (asasmith) [inline web/src/pages/404.js]: Please rename this `bodyImage`.

**Review by asasmith — CHANGES_REQUESTED** [review]

File: web/src/pages/404.js (hunk 3 above)
Comment (asasmith) [reply web/src/pages/404.js]: Actually, I'm not sure the negative margin is necessary. Let's leave it out for now.

Comment (kcbreuil) [reply web/src/pages/404.js]: @asasmith the hero component has a bottom margin of 10rem. the designs don't call for the large of a space between the top component and the 404 section. I added it to match the designs. if not it looks weird.

Comment (asasmith) [reply web/src/pages/404.js]: I didn't see a significant difference. If you think it's necessary please move it to a tailwind class.

File: web/src/pages/404.js [hunk 7]
```diff
@@ -1,63 +1,91 @@
...
-const Main = styled.main`
-    ${tw`container`}
+import Layout from '../page-layouts/layout';
+import Hero from '../components/hero';
+import CtaSection from '../components/ctaSection';
 
-    color: '#232129',
-    fontFamily: '-apple-system, Roboto, sans-serif, serif',
+const Wrapper = styled.div`
+    ${tw`flex flex-col justify-center items-center content-center`}
```
Comment (asasmith) [inline web/src/pages/404.js]: Should probably add the `container` class here.

File: web/src/pages/404.js [hunk 8]
```diff
@@ -1,63 +1,91 @@
...
-//     color: '#232129',
-//     fontFamily: '-apple-system, Roboto, sans-serif, serif',
-// };
-const headingStyles = {
-    marginTop: 0,
-    marginBottom: 64,
-    maxWidth: 320,
-};
+const StyledHeading = styled.h2`
+    ${tw`font-bold text-center text-5xl w-1/2 mb-8`}
```
Comment (asasmith) [inline web/src/pages/404.js]: `w-1/2` doesn't work well on small screens.

File: web/src/pages/404.js [hunk 9]
```diff
@@ -1,63 +1,91 @@
...
-    borderRadius: 4,
-};
+    line-height: 3.75rem;
+    max-width: 58rem;
+`;
+
+const StyledP = styled.p`
+    ${tw`text-2xl mb-12`}
+
+    line-height: 150%;
```
Comment (asasmith) [inline web/src/pages/404.js:23]: Can we use a tailwind class for this?

File: web/src/pages/404.js [hunk 10]
```diff
@@ -1,63 +1,91 @@
...
+`;
+
+const StyledLink = styled(Link)`
+    ${tw`text-green`}
+`;
 
 // markup
```
Comment (asasmith) [inline web/src/pages/404.js]: remove this comment

**Review by asasmith — CHANGES_REQUESTED** [review]

**Review by asasmith — APPROVED** [review]

**Pull request merged**